docs/Capabilities/Functionalities/Importing-and-exporting-form-designs.md

```diff
@@ -33,5 +33,16 @@ To export a form:
 
 Exporting forms is a great way to copy an existing design or share it with others.
 
+
+## Creating a form using JSON with minimum fields.
+CHEFS Form.io platform provides a visual form builder that lets you drag and drop form elements, and automatically generates the corresponding JSON schema for you. Every component and layouts in CHEFS are represented as JSON in the background.
+
+If for any reason if you would like to create the forms using the JSON Schema instead of visual form builder, below are some JSON schemas for some components with minimum mandatory variables and full schema.
```

Comment:
```suggestion
If for any reason you would like to create the forms using the JSON Schema instead of the visual form builder, below are some JSON schemas for some components with minimum mandatory variables and full schema.
```